.github/workflows/ci-dgraph-tests.yml

```diff
@@ -60,7 +60,7 @@ jobs:
           # build the test binary
           cd t; go build .
           # run the tests
-          ./t --coverage=true --skip systest/backup,systest/online-restore,systest/loader,tlstest
+          ./t --skip systest,dgraph/cmd,graphql/e2e
```

Comment:
@SashaDgraph your rebase seems to incorrect here, we have opened up all the tests. And we have left out specific subfolders in `systest`. You will have to align this file correctly to point to this https://github.com/dgraph-io/dgraph/blob/main/.github/workflows/ci-dgraph-tests.yml#L63

This should ideally get resolved with a rebase* - could you please see if you can fix this up?

Reply:
OK. I will look into it and update in a few minutes.

Reply:
Updated. Please review when tests are done running.

Reply:
Thanks 🙏